Orion/Orion/API/APIManager.m

```diff
@@ -80,11 +80,36 @@ - (void)getCategorySpecificTrending:(NSDictionary *)passedCategoryInfo completio
         [task resume];
 }
 
+- (void)getSearching:(NSString *)passedSearchWord completion:(void(^)(NSArray *products))completion {
+    NSString *searchParam = [NSString stringWithFormat:@"%@%@%@%@%@", productAPIURLString, @"(name=", passedSearchWord, @"*)?format=json&show=sku,name,regularPrice,image&pageSize=10&apiKey=" ,self.api_key];
```

Comment:
since the list of products comes back in pages, we should support pulling the next page when the user reaches the end of the list! Otherwise they'll only ever see the top 10 results for the search